pages/api/sendConfirmationEmail.ts

```diff
@@ -0,0 +1,83 @@
+import nodemailer from 'nodemailer'; // Nodemailer
```

Comment:
In Next.js, the `pages/api` folder is reserved for API routes. This confirmation email util is just a helper, so it should go in `lib/reports/utils.ts`

Reply:
My bad! Was supposed to put it in the api folder temporarily until I figure out the correct folder but forget it there! Resolved!